draft-ietf-httpapi-ratelimit-headers.md

```diff
@@ -414,6 +414,19 @@ moment.
 Nonetheless servers MAY decide to send the `RateLimit` fields
 in a trailer section.
 
+## Performance considerations
+
+Servers are not required to return `RateLimit` fields
```

Comment:
@unleashed wdyt?

Reply:
I thought this is already assumed since we make the headers and values independent between requests and they are optional anyway, but I think it's good to explicitly call this out.